sdk/nextjs/src/pages/bucketing.ts

```diff
@@ -0,0 +1,52 @@
+import { DevCycleUser, DVCPopulatedUser } from '@devcycle/js-client-sdk'
+import { generateBucketedConfig } from '@devcycle/bucketing'
+import { sseURlGetter } from '../server/ably'
+import { BucketedUserConfig } from '@devcycle/types'
+
+const getFetchUrl = (sdkKey: string) =>
+    `https://config-cdn.devcycle.com/config/v1/client/${sdkKey}.json`
+
+export const fetchCDNConfig = async (sdkKey: string): Promise<Response> => {
+    return await fetch(getFetchUrl(sdkKey))
```

Comment:
it doesn't make a functional difference, it's more of a stylistic choice. I prefer to explicitly write `await` because it makes it easier to understand that the call is async